sympy/stats/drv_types.py

```diff
@@ -367,6 +368,97 @@ def Poisson(name, lamda):
     return rv(name, PoissonDistribution, lamda)
 
 
+# -----------------------------------------------------------------------------
+# Skellam distribution --------------------------------------------------------
+
+
+class SkellamDistribution(SingleDiscreteDistribution):
+    _argnames = ('mu1', 'mu2')
+    set = S.Integers
+
+    @staticmethod
+    def check(mu1, mu2):
+        _value_check(mu1 >= 0, 'Parameter mu1 must be >= 0')
+        _value_check(mu2 >= 0, 'Parameter mu2 must be >= 0')
+
+    def pdf(self, k):
+        mu1, mu2 = self.mu1, self.mu2
+        term1 = exp(-(mu1+mu2))*(mu1/mu2)**(k/2)
+        term2 = besseli(k, 2*sqrt(mu1*mu2))
+        return term1*term2
+
+    def _cdf(self, x):
+        return NotImplementedError("Skellam doesn't have closed form for the CDF.")
+
+    def _characteristic_function(self, t):
+        mu1, mu2 = self.mu1, self.mu2
+        return exp(-(mu1+mu2)+mu1*exp(t)+mu2*exp(-t))
+
+    def _moment_generating_function(self, t):
+        mu1, mu2 = self.mu1, self.mu2
+        return exp(-(mu1+mu2)+mu1*exp(I*t)+mu2*exp(-I*t))
+
+
+def Skellam(name, mu1, mu2):
+    r"""
+    Create a discrete random variable with a Skellam distribution.
+
+    The Skellam is the distribution of the difference N1 - N2
+    of two statistically independent random variables N1 and N2
+    each Poisson-distributed with respective expected values mu1 and mu2.
+
+    The density of the Skellam distribution is given by
+
+    .. math::
+        f(k) := e^{-(\mu_1+\mu_2)}(\frac{\mu_1}{\mu_2})^{k/2}I_k(2\sqrt{\mu_1\mu_2})
+
+    Parameters
+    ==========
+
+    mu1: A non-negative value
+    mu2: A non-negative value
+
+    Returns
+    =======
+
+    A RandomSymbol.
+
+    Examples
+    ========
+
+    >>> from sympy.stats import Skellam, density, E, variance
+    >>> from sympy import Symbol, simplify, pprint
+
+    >>> z = Symbol("z", integer=True)
+    >>> mu1 = Symbol("mu1", positive=True)
+    >>> mu2 = Symbol("mu2", positive=True)
+
+    >>> X = Skellam("x", mu1, mu2)
+
+    >>> pprint(density(X)(z), use_unicode=False)
+         z
+         -
+         2
+    /mu1\   -mu1 - mu2        /       _____   _____\
+    |---| *e          *besseli\z, 2*\/ mu1 *\/ mu2 /
+    \mu2/
+
+    >>> E(X)
+    I*mu1 - I*mu2
+
+    >>> simplify(variance(X))
+    -mu1 - mu2
+
+    References
+    ==========
+
+    .. [1] https://en.wikipedia.org/wiki/Skellam_distribution
+    .. [2]
```

Comment:
Did you intend to add another reference?

Reply:
No